website/source/docs/concepts/dev-server.html.md

```diff
@@ -46,6 +45,9 @@ The properties of the dev server:
 
 The dev server should be used for experimentation with Vault features, such
 as different authentication backends, secret backends, audit backends, etc.
+If you're new to Vault, you may want to pick up with [Your First 
+Secret](http://vaultproject.io/intro/getting-started/first-secret.html) in 
```

Comment:
This should be https, but I am going to merge as-is and do a pass across all the docs :smile: